hack/release/prepare-assets.sh

```diff
@@ -107,6 +107,8 @@ export IMG_TAG=$VERSION
 export IMG_NAME=projects.registry.vmware.com/antrea/antrea-ubuntu
 ./hack/generate-standard-manifests.sh --mode release --out "$OUTPUT_DIR"
 
+sed "s|antrea\/antrea-ubuntu:latest|$IMG_NAME:$VERSION|g" ./build/yamls/antrea-migrator.yml > "$OUTPUT_DIR"/antrea-migrator.yml
```

Comment:
This is wrong. The $IMG_NAME here is "projects.registry.vmware.com/antrea/antrea-ubuntu", but you are using migrator image in the migrator.yml. You should replace "antrea/antrea-migrator:latest" with `projects.registry.vmware.com/antrea/antrea-migrator:$VERSION`
Please make sure you verify this with the new manifest.